modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala

```diff
@@ -183,9 +186,35 @@ trait VoTableParser {
 
     val parseMagnitudes = adapter.parseMagnitudes(entries)
 
+    def parseAngularSize: ValidatedNec[CatalogProblem, Option[AngularSize]] = {
+
+      def parseDoubleMinutesOpt(field: FieldId): ValidatedNec[CatalogProblem, Option[Angle]] =
+        entries.get(field) match {
+          case Some(v) =>
+            parseDoubleValue(field.ucd, v)
+              .map(_ * 60 * 1e6)
```

Comment:
Is it coming as a double in degrees? Maybe we can parse directly?

Reply:
It's coming in as double in minutes.

Reply:
Why make it easy, right

Reply:
I'm adding a comment